test/services/active.officers.details.service.unit.ts

```diff
@@ -0,0 +1,90 @@
+jest.mock("@companieshouse/api-sdk-node");
+jest.mock("@companieshouse/api-sdk-node/dist/services/confirmation-statement");
+
+import { createApiClient, Resource } from "@companieshouse/api-sdk-node";
+import { getActiveOfficersDetailsData } from "../../src/services/active.officers.details.service";
+import { mockActiveOfficersDetails, mockActiveOfficersDetailsFormatted } from "../mocks/active.officers.details.mock";
+import { getSessionRequest } from "../mocks/session.mock";
+import { ApiErrorResponse } from "@companieshouse/api-sdk-node/dist/services/resource";
+import { formatOfficerDetails } from "../../src/utils/format";
+import ApiClient from "@companieshouse/api-sdk-node/dist/client";
+import {
+  ActiveOfficerDetails,
+  ConfirmationStatementService
+} from "@companieshouse/api-sdk-node/dist/services/confirmation-statement";
+
+const mockGetActiveOfficersDetails = ConfirmationStatementService.prototype.getListActiveOfficerDetails as jest.Mock;
+const mockCreatePrivateApiClient = createApiClient as jest.Mock;
+
+mockCreatePrivateApiClient.mockReturnValue({
+  confirmationStatementService: ConfirmationStatementService.prototype
+} as ApiClient);
+
+const clone = (objectToClone: any): any => {
+  return JSON.parse(JSON.stringify(objectToClone));
+};
+
+describe("Test active officers details service", () => {
+
+  const TRANSACTION_ID = "66454";
+  const SUBMISSION_ID = "435435";
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("Should call the sdk and get the active officers details data", async () => {
+
+    const resource: Resource<ActiveOfficerDetails> = {
+      httpStatusCode: 200,
+      resource: mockActiveOfficersDetails
+    };
+
+    mockGetActiveOfficersDetails.mockReturnValueOnce(resource);
```

Comment:
for the get test, as active.officers.details.service returns an array of ActiveOfficerDetails, i'd change the test so that the resource is an array (of 2?) and test that the call to getActiveOfficersDetailsData returns the 2 items (just to test it handles > 1 item)